src/Tasks.UnitTests/CreateCSharpManifestResourceName_Tests.cs

```diff
@@ -266,6 +266,29 @@ public void RootnamespaceWithCulture()
             Assert.Equal("RootNamespace.SubFolder.MyForm.en-GB", result);
         }
 
+        /// <summary>
+        /// Explicitly retain culture
+        /// </summary>
+        [Fact]
+        public void RootnamespaceWithCulture_RetainCultureInFileName()
+        {
+            string result =
+            CreateCSharpManifestResourceName.CreateManifestNameImpl
+                (
+                    @"Subfolder\File.cs.cshtml",
```

Comment:
I have a vague recollection that you were going to add the argument names here to make it clearer?